### ISSUE TITLE
请教问题

### ISSUE THREAD

**Issue by HejhGitHub** [klintan/pytorch-lanenet#23]
decode_logits = decode_ret['logits']
decode_deconv = decode_ret['deconv']
请问这两行代码是什么意思？学习
